docs/sources/administration/plugin-management/index.md

```diff
@@ -80,91 +88,91 @@ The Plugin catalog allows you to browse and manage plugins from within Grafana.
 
 _Video shows the Plugin catalog in a previous version of Grafana._
 
-In order to be able to install / uninstall / update plugins using plugin catalog, you must enable it via the `plugin_admin_enabled` flag in the [configuration]({{< relref "../../setup-grafana/configure-grafana/#plugin_admin_enabled" >}}) file.
-Before following the steps below, make sure you are logged in as a Grafana administrator.
+In order to be able to install, uninstall, or update plugins using the plugin catalog, you must enable it with the `plugin_admin_enabled` flag in the [configuration]({{< relref "../../setup-grafana/configure-grafana/#plugin_admin_enabled" >}}) file.
```

Comment:
```suggestion
```

Reply:
Agree on the changes due to the fact:

1. It is already enabled by default.
2. So, far in my experience while working with users, no one come and asked as how to disable this specific part.